docs/docusaurus/docs/components/warnings/_cli_removal.md

```diff
@@ -0,0 +1,5 @@
+:::caution Update in progress
+
+The Great Expectations CLI is deprecated. This topic will be updated soon to reflect this functionality change.
```

Comment:
- I would only like this banner to go live after we publish the blog + community message. Thus, this PR should link to the blog.
- We're not actually deprecating the CLI since it's still functional, just removing references in the docs.
- Should we say "this page" instead of "this topic"?
- Since we're not actually deprecating the CLI, I think we'll have to rewrite this sentence.

Reply:
@joshzzheng 

- The original draft of the admonition I created this morning included a link to the blog. It was removed during the review process. The link is included in the latest draft of the admonition. 
- I removed _deprecating_ from the latest draft of the admonition. I requested your review of the new text [here](https://greatexpectationslabs.slack.com/archives/C0546LBAK0E/p1682701745543389).
- Typically, _page_ is reserved for "references to web pages and sub-pages of consoles". The admonition appears within topics in the help. So, _topic_ is correct here. 
- I have rewritten the admonition and requested your review [on Slack](https://greatexpectationslabs.slack.com/archives/C0546LBAK0E/p1682701745543389).